task-sdk/src/airflow/sdk/definitions/dag.py

```diff
@@ -1096,10 +1096,10 @@ def test(
         from contextlib import ExitStack
 
         from airflow import settings
-        from airflow.configuration import secrets_backend_list
         from airflow.models.dag import DAG as SchedulerDAG, _get_or_create_dagrun
         from airflow.models.dagrun import DagRun
         from airflow.sdk import timezone
+        from airflow.sdk._shared.configuration import secrets_backend_list
```

Comment:
In some place we use:

```py
from airflow.sdk.configuration import conf
```

and here:

```
from airflow.sdk._shared.configuration import secrets_backend_list
```

We should use one or the other imo consistently.
`airflow.sdk._shared.configuration` vs `from airflow.sdk.configuration`

Reply:
That, I kind of did intentionally. I have added a new module(task-sdk/src/airflow/sdk/configuration.py_) which can be a public facing API in case users / dag authors need to use it in dags etc, they could do `airflow.sdk.configuration.conf` but for anything else, which is not consumed by end users / dag authors, real path has to be used/